### PR TITLE
gravatar: fixed special chars decoding in bio

### LINKED ISSUE #2676: Gravatar: Encoding bug
Im not sure exactly what the issue is;
If you search up Gravatar x@gmail.com and you have & in your bio is displays as &amp; .

------
IA Page:  http://duck.co/ia/view/gravatar 
[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @adman

### PR DESCRIPTION
**Pull request by Adman**
###### What does your Pull Request do (check all that apply)?

Choose the most relevant items and use the following title template to name
your Pull Request.

- [ ] New Instant Answer: **`New {IA Name} Instant Answer`**
- [x] Improvement
    - [x] Bug fix: **`{IA Name}: Fix {Issue number or one-line description}`**
    - [ ] Enhancement: **`{IA Name}: {Description of Improvements}`**
- [ ] Non–Instant Answer
    - [ ] Other (Role, Template, Test, Documentation, etc.): **`{SpiceRole/Templates/Tests/Docs}: {Short Description}`**

###### Description of changes

Added function for converting special chars into HTML.
Not sure whether creating HTML element and pasting-cutting text from it is the best way of doing this, so I would appreciate any idea which would solve this problem in a nicer way.

###### Which issues (if any) does this fix?

Fixes  #2676

###### People to notify (@mention interested parties)
to @moollaza 

---

IA Page: http://duck.co/ia/view/gravatar

[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @adman

Signed-off-by: Adrian Matejov <a.matejov@centrum.sk>

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/spice/gravatar/gravatar.js [hunk 1]
```diff
@@ -71,12 +71,19 @@
...
+                    return elem.value;
+                }
+
                 return {
                     image: api_result.entry[0].thumbnailUrl + ".png",
                     title: getName(api_result.entry[0]),
                     subtitle: [getUsername(api_result.entry[0]), getCurrentLocation(api_result.entry[0])],
                     altSubtitle: getAccounts(api_result.entry[0]),
-                    description: api_result.entry[0].aboutMe
+                    description: decodeSpecialChars(api_result.entry[0].aboutMe)
```
Comment (moollaza) [inline share/spice/gravatar/gravatar.js]: We have a `DDG.unescape()` function, please use that as it handles the content using a safer approach

https://github.com/duckduckgo/duckduckgo-utils/blob/master/util.js#L653

**Comment by Adman** [comment]
@moollaza Should be fixed

**Comment by edgesince84** [comment]
@Adman thanks for the PR, you move quick! ⏩

**Comment by moollaza** [comment]
Deploying to Beta...

**Comment by moollaza** [comment]
@Adman Thanks! This LGTM :+1:

https://beta.duckduckgo.com/?q=gravatar+simonfolde%40gmail.com&ia=avatar

**Comment by edgesince84** [comment]
@Adman thanks again! 🏃

**Pull request merged**